### PR TITLE
Added a chat content background that respects dark mode

### LINKED ISSUE #5: Chat window surface color doesn't respect light/dark theme
light-dark-mode

### PR DESCRIPTION
**Pull request by dgmltn**
This fixes #5

### CODE REVIEW

File: getting-started/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt [hunk 1]
```diff
@@ -35,7 +37,9 @@ object Chat {
         val localFocusManager = LocalFocusManager.current
 
         Column(
-            modifier = Modifier.fillMaxSize()
+            modifier = Modifier
+                .fillMaxSize()
+                .background(MaterialTheme.colors.surface)
```
Comment (Chesteer89) [inline getting-started/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt]: I'd rather use `background` color. 

A brief explaination:
`Surface colors map to components such as cards, sheets, and menus`
`Background color is found behind scrollable content`
source: [Material Theming](https://material.io/design/material-theming/implementing-your-theme.html#color)

**Review by Chesteer89 — CHANGES_REQUESTED** [review]
Hi! 
Thanks for your contribution. Please take a look at the comment about the used color.

I would also like to inform you that we are working on the default appearance of the components that will fix this issue. Until then, we can merge this temporary fix.

**Comment by dgmltn** [comment]
Thanks for the explanation! My PR is updated with `background` instead of `surface`

File: getting-started/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt (hunk 1 above)
Comment (dgmltn) [reply getting-started/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt]: Change pushed ✔️

**Review by Chesteer89 — APPROVED** [review]
LGTM!

**Pull request merged**